### PR TITLE
PLAT-54: Login Component should route to the redirect URL on log in

### PR DESCRIPTION
**Pull request by dondrzzy**
## JIRA TICKET NAME
[PLAT-54: Login Component should route to the redirect URL on log in](https://issues.openmrs.org/browse/PLAT-54)

## Description of what I changed

- Route users on login to the redirect URL if it exists.

## Issue I worked on
[PLAT-54](https://issues.openmrs.org/browse/PLAT-54)

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

  No? -> [read here](https://wiki.openmrs.org/display/docs/Pull+Request+Tips) on how to squash multiple commits into one

- [x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [x] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [ ] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [x] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [x] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

**Review by muhozi — APPROVED** [review]
LGTM

File: app/js/components/login/Login.jsx [hunk 1]
```diff
@@ -53,9 +53,17 @@ class Login extends Component {
   )
 
   handleResponse = (response) => {
-    response.data.authenticated
-      ? this.props.history.push('/')
-      : this.toastError('Incorrect username or password');
+    const redirectUrl = window.location.href.split('redirect=')[1] || null;
+    if (response.data.authenticated) {
+      if (redirectUrl) {
+        console.log('redirectUrl', redirectUrl);
```
Comment (Daramola98) [inline app/js/components/login/Login.jsx]: Please remove the `console.log()` if it no longer serves any purpose

**Review by Daramola98 — APPROVED** [review]
LGTM

**Review by Efosaok — APPROVED** [review]
LGTM

**Review by emmabaye — APPROVED** [review]
LGTM

**Review by IAMOTZ — APPROVED** [review]

**Comment by dkayiwa** [comment]
Can you look into these merge conflicts?

**Comment by dondrzzy** [comment]
Okay

**Comment by dondrzzy** [comment]
Addressed

File: app/js/components/login/Login.jsx (hunk 1 above)
Comment (dondrzzy) [reply app/js/components/login/Login.jsx]: Addressed

**Pull request merged**